apps/console/src/extensions/components/authenticators/sms-otp/sms-otp-authenticator.tsx

```diff
@@ -78,14 +80,32 @@ export const SmsOTPAuthenticator: FunctionComponent<SmsOTPAuthenticatorInterface
         ...rest
     } = props;
 
-    const [ isReadOnly, setIsReadOnly ] = useState<boolean>(true);
+    const featureConfig: FeatureAccessConfigInterface = useSelector((state: AppState) => {
+        return state.config?.ui?.features?.smsProviders;
+    });
+
+    const isChoreoEnabledAsSMSProvider: boolean = useMemo(() => {
+        const disabledFeatures: string[] = featureConfig?.disabledFeatures;
+
+        return !disabledFeatures.includes("choreoAsSMSProvider");
```

Comment:
Null check needs to be added here. If the feature config is not defined in the deployment config, `disabledFeatures` will be `undefined`